cishouseholds/pipeline/survey_responses_version_digital_ETL.py

```diff
@@ -241,7 +242,17 @@ def digital_specific_transformations(df):
     )
     df = clean_barcode_simple(df, "swab_sample_barcode_user_entered")
     df = clean_barcode_simple(df, "blood_sample_barcode_user_entered")
-
+    df = map_options_to_bool_columns(
+        df,
+        "currently_smokes_or_vapes_description",
+        {
+            "cigarettes": "smoke_cigarettes",
+            "cigars": "smokes_cigar",
```

Comment:
some of these cases exist where multiple options are selected and ; separated  therefore we need to split these strings out as an array column first then use the contents of the array to populate remaining columns